dali/base/dadfs.cpp

```diff
@@ -7929,6 +7956,14 @@ IDistributedSuperFile *CDistributedFileDirectory::createSuperFile(const char *_l
 }
 
 // MORE: This should be implemented in DFSAccess later on
+IDistributedSuperFile *CDistributedFileDirectory::createNewSuperFile(ISuperFileDescriptor *desc, IUserDescriptor *user, bool includeports)
+{
+    Owned<IDistributedSuperFile> superfile;
+    superfile.setown( new CDistributedSuperFile(this, desc, user));
+    return superfile.getLink();
```

Comment:
no other paths and not doing anything else with object 'superfile', so simpler as:
```
return new CDistributedSuperFile(this, desc, user);
```